src/smiley_extractor.py

```diff
@@ -0,0 +1,69 @@
+import json
+from xml.etree import ElementTree as ET
+from requests import get
+
+
+class SmileyExtractor:
+    """
+    Class responsible for extracting data from the smiley XML file
+    """
+    SMILEY_XML_URL = 'https://www.foedevarestyrelsen.dk/_layouts/15/sdata/smiley_xml.xml'
+    SMILEY_XML = 'smiley_xml.xml'
+    SMILEY_JSON = 'smiley_json.json'
+
+    @staticmethod
+    def create_smiley_json() -> None:
+        """
+        Create .json file from smiley XML data from Fødevarestyrelsen.
+        """
+        SmileyExtractor._retrieve_smiley_data()
+
+        tree = ET.parse(SmileyExtractor.SMILEY_XML)
+        root = tree.getroot()
+
+        res = []
+
+        for row in list(root):
+            new_obj = {col.tag: col.text for col in row}
+            SmileyExtractor._convert_to_float(new_obj, 'Geo_Lat', 'Geo_Lng')
+            SmileyExtractor._convert_to_int(new_obj, 'seneste_kontrol', 'naestseneste_kontrol',
+                                            'tredjeseneste_kontrol', 'fjerdeseneste_kontrol')
+            SmileyExtractor._strip_whitespace(new_obj, 'navn1')
+            res.append(new_obj)
+
+        with open(SmileyExtractor.SMILEY_JSON, 'w') as f:
+            f.write(json.dumps(res, indent=4, sort_keys=True))
```

Comment:
Why is this static? Induces unnecessary calls through the class rather than the instance

Reply:
If you want them to be callable without creating an instance, make this a class method.